src/Symfony/Component/DomCrawler/Form.php

```diff
@@ -21,19 +21,19 @@
  */
 class Form extends Link implements \ArrayAccess
 {
-    private \DOMElement $button;
-    private FormFieldRegistry $fields;
-    private ?string $baseHref;
+    private \DOMElement|\DOM\Element $button;
+    protected FormFieldRegistry $fields;
+    protected ?string $baseHref;
```

Comment:
Could we find a way to avoid turning private properties into protected ones (which become covered by the BC promise) ?

Reply:
That's a mistake from an old try, reverting